extensions/cpsection/frame/model.py

```diff
@@ -67,3 +67,32 @@ def set_edge_delay(delay):
     settings = Gio.Settings('org.sugarlabs.frame')
     settings.set_int('edge-delay', int(delay))
     return 0
+
+
+def get_trigger_size():
+    settings = Gio.Settings('org.sugarlabs.frame')
+    edge_delay = settings.get_int('edge-delay')
+    return edge_delay
```

Comment:
get_trigger_size reads a setting 'edge-delay', perhaps this should be 'trigger-size'?